test/project.spec.js

```diff
@@ -23,7 +23,7 @@ describe('Project', function() {
       github
         .getUser()
         .createRepo({name: testRepoName})
-        .then(wait())
+        .then(wait(5000))
```

Comment:
I generally find arbitrary delays like this in tests to be brittle. Perhaps a better solution is to poll for the repo until it comes back successfully.